test/test_torch.py

```diff
@@ -15149,6 +15149,21 @@ def test_full_out(self, device):
         self.assertEqual(torch.full(o.shape, 1., out=o).dtype, o.dtype)
 
 
+    @onlyOnCPUAndCUDA
+    def test_complex_type_conversions(self, device):
```

Comment:
can we add this test here: https://github.com/pytorch/pytorch/blob/361eed6a6e085856ac8a6877337a4d46fe8a7fb6/test/test_torch.py#L1144 so that copy behavior for all dtypes is tested in one test function?

Reply:
I'd prefer not to do it because
1) `test_to` tests different things (whether tensor is actually copied or not depending on the `to` parameters), not the correctness of conversions with the different dtypes
2) those tests are not using the new-style testing framework, and we are trying not to expand that section. 
This test is sufficiently different. If you prefer, I can expand it to test wider matrix of conversions, but it is still a separate test from the one you pointed out, and it should remain in TestTorchDeviceType.

Reply:
ok let's keep it as is then! thank you:)